rules-java/api/src/main/java/org/jboss/windup/rules/apps/java/reporting/rules/CreateJavaApplicationOverviewReportRuleProvider.java

```diff
@@ -82,12 +84,17 @@ public String toString()
 
     private void createApplicationReport(GraphContext context, ProjectModel projectModel)
     {
-        createMainApplicationOverviewReport(context, projectModel, 100, false, DETAILS_REPORT, Collections.EMPTY_SET, Collections.singleton(TAG_CATCHALL));
-        createMainApplicationOverviewReport(context, projectModel, 101, false, CATCHALL_REPORT, Collections.singleton(TAG_CATCHALL),
-                    Collections.EMPTY_SET);
+        createMainApplicationOverviewReport(context, projectModel, APPLICATION_DETAIL_DESCRIPTION, 101, false, DETAILS_REPORT,
+                    Collections.<String> emptySet(),
+                    Collections.singleton(TAG_CATCHALL));
+        createMainApplicationOverviewReport(context, projectModel,
+                    CATCHALL_REPORT_DESCRIPTION,
+                    102, false, CATCHALL_REPORT, Collections.singleton(TAG_CATCHALL),
+                    Collections.<String> emptySet());
     }
 
-    private void createMainApplicationOverviewReport(GraphContext context, ProjectModel projectModel, int priority, boolean main, String name,
+    private void createMainApplicationOverviewReport(GraphContext context, ProjectModel projectModel, String description, int priority, boolean main,
```

Comment:
The "main" part is now redundant as neither of them are the main report. Main indicates that this is the report to link to from the application list.